nucypher/network/server.py

```diff
@@ -422,30 +422,28 @@ def receive_treasure_map():
 
     @rest_app.route('/status/', methods=['GET'])
     def status():
-        if request.args.get('json'):
-            payload = this_node.abridged_node_details(raise_invalid=False)
-            response = jsonify(payload)
-            return response
+
+        return_json = request.args.get('json') == 'true'
```

Comment:
Why check for `true` explicitly here - to avoid the case of evaluating `false` as True?  I find this to be a bit awkward, perhaps we need to consider using an alternate endpoint?

Reply:
Isn't that the correct way to do it? As far as I understand, URL arguments are given to the request object as strings, casting them is the user's responsibility.

Reply:
Yeah, alright, fair enough.